testing/tests/index.test.js

```diff
@@ -201,3 +205,78 @@ test("check built flaws for /en-us/learn/css/css_layout/introduction/grid page",
   const { doc } = JSON.parse(fs.readFileSync(jsonFile));
   expect(doc.flaws.macros.length).toBe(3);
 });
+
+describe("flaw-fixing tests", () => {
+  beforeAll(() => {
+    execSync("node content build --fix-flaws", {
```

Comment:
Of course, this doesn't work without your flaw-fixing PR. I'm just showing it as an example.